app/src/main/java/com/github/onlynotesswent/model/folder/FolderViewModel.kt

```diff
@@ -364,6 +379,28 @@ class FolderViewModel(private val repository: FolderRepository) : ViewModel() {
           useCache = useCache)
     }
   }
+  /**
+   * Updates an existing folder without changing the state of the ViewModel.
+   *
+   * @param folder The folder with updated information.
+   * @param onSuccess The function to call when the folder is updated successfully.
+   * @param onFailure The function to call when the folder fails to be updated.
+   * @param useCache Whether to update data from cache. Should be true only if userId of the folder
+   *   is the current user.
+   * @param isDeckView A flag indicating if the folder is a deck view.
+   */
+  fun updateFolderNoStateUpdate(
+      folder: Folder,
+      onSuccess: () -> Unit = {},
+      onFailure: (Exception) -> Unit = {},
+      useCache: Boolean = false,
+      isDeckView: Boolean = false
```

Comment:
Do we need this parameter if it is not getting used?

Reply:
Yes good point I was copying the parameters of the corresponding methods with the NoStateUpdate but here the parameter is useless. I removed it.